Coucat/Pokedex
# Feature/pok 22 create search nav

# Pull Request

## Description

### Ticket impemented #[POK-22](https://coucat.atlassian.net/browse/POK-22)

[//]:<> (You can see the number of ticket in Jira in the top of task, replace the ticketNumber in the space and the number in the link)
### What did you implement it?

[//]:<> (Mark with an 'x' or a space)



### How did you implement it?
[//]:<> (In this section describe all things that you made)
The search nav was created in two parts 

1. Input and button to search

* This part is composed of a div, an input and a button, the div contains the input and the button. The button only has a function to send what is in the input and the filters 

2. Filters and dropdown

* The dropdowns are composed of several labels, it is not a simple component and it has the particularity of placing icons on the items if they are necessary. They also have functions to mark the items if they are clicked as well as the dropdown. 

* The filters work together with the dropdown when you select an item from the dropdown it is sent to the filters and the button to delete all is activated.  

* Also within this an index was created that contains all the icons in svg format so that they can be used more easily
## How can we test this ticket?
[//]:<> (Write in this part all the commands that you can use to test your code)
To test the delete button you have to go to the search page and place some word in the input that says "search your pokemon" and then select an item from the dropdowns after this you can try the clear filters button and see how it resets the dropdowns.
To test the filters you must do the same as in the previous one but using the search button and to see that it worked you need to press _F12_ and search for the console tab and there you will see an object that contains everything placed in the filters

[//]:<> (Mark with a 'x' or a space )
